pkg/service/nodes.go

```diff
@@ -75,18 +76,23 @@ func runCommandWithContext(call string, args []string, timeoutSeconds, retryInte
 	defer cancel()
 	err := retry.Do(
 		func() error {
+			log.Info("[sbadiger] about to excute command: ", call, " args: ", args)
 			cmd := exec.CommandContext(ctx, call, args...)
 			_, err := cmd.CombinedOutput()
 			if err != nil {
+				log.Info("[sbadiger] command execution failed: ", call, " args: ", args, " error: ", err)
```

Comment:
clean the log info please

Reply:
lol. Missed it. Thanks.

Reply:
Looks like I created the PR from the wrong branch. I will collapse the branch and open the PR for review.